### ISSUE TITLE
[🧐 Evaluating] 🐞 Bug Report: [i18n] Some texts of the Admin Top Bar are not displayed translated

### ISSUE THREAD

**Issue by DAnn2012** [elementor/elementor#22388]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Some texts of the Admin Top Bar are not displayed translated if the language is not English, for example: "**Search or do anything in Elementor**", "**Connect Account**" and "**Connect your account to get access to Elementor's Template Library & more.**"

![elem1](https://github.com/elementor/elementor/assets/1197819/60e54a0a-a332-4759-adbf-9f60841f77cf)
![elem2](https://github.com/elementor/elementor/assets/1197819/868a61fd-0c8a-4c70-9430-dc82e2849ec4)

Also, "`&#8984;`" is shown on Mac and not the Command key symbol (⌘).

### Steps to reproduce

With the language set to, for example, Spanish or Italian, go to the WordPress Dashboard and select the "Elementor" menu item

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: TasteWP-S3 Official/3.0.0
	MySQL version: (Ubuntu) v8.0.32
	PHP Version: 7.4.33
	PHP Memory Limit: 
	PHP Max Input Vars: 5000
	PHP Max Post Size: 60M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: https://accessibleproperty.s3-tastewp.com
	Home URL: https://accessibleproperty.s3-tastewp.com
	WP Multisite: No
	Max Upload Size: 60 MB
	Memory limit: 96M
	Max Memory limit: 96M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: es
	Timezone: 0
	Admin Email: accessibleproperty@tastewp.com
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: it_IT
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.4.1 Safari/605.1.15

== Active Plugins ==
	Elementor
		Version: 3.13.1
		Author: Elementor.com


== Experimentos de Elementor ==
	Salida del DOM optimizada: Activo por defecto
	Carga de recursos mejorada: Activo por defecto
	Carga de CSS mejorada: Activo por defecto
	Fuentes de iconos integradas: Inactivo por defecto
	Mejoras de accesibilidad: Activo por defecto
	Puntos de ruptura personalizados adicionales: Activo por defecto
	admin_menu_rearrangement: Inactivo por defecto
	Contenedor Flexbox: Inactivo por defecto
	Actualizar la librería Swiper: Activo por defecto
	Contenedor Grid: Inactivo por defecto
	Cabecera y pie de página del tema Hello: Activo por defecto
	Barra superior del editor: Inactivo por defecto
	Páginas de destino: Activo por defecto
	Elementos anidados: Inactivo por defecto
	Carga diferida de imágenes de fondo: Inactivo por defecto


== Registro ==
	
PHP: showing 1 of 1PHP: 2023-05-09 14:03:52 [warning X 1][/s3-accessibleproperty/wordpress/wp-content/plugins/elementor/modules/system-info/reporters/server.php::157] get_cfg_var() has been disabled for security reasons [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	

				
````

</details>

**Comment by nicholaszein** [comment]
Hello @DAnn2012! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hello @DAnn2012!

This issue was solved with your PR and merged into the 3.15.2 release.

Thank you so much for your contribution! 🙏 

Kind regards
